extensions/blocks/podcast-player/edit.js

```diff
@@ -106,6 +116,14 @@ const PodcastPlayerEdit = ( {
 		[ createErrorNotice ]
 	);
 
+	useEffect( () => {
+		return () => {
+			if ( cancellableFetch && cancellableFetch.current && cancellableFetch.current.cancel ) {
+				cancellableFetch.current.cancel();
+			}
+		};
+	}, [] );
+
```

Comment:
Simple suggestion to make it clearer. Feel free to ignore it :-)

```suggestion
	useEffect( () => (
		() => get( cancellableFetch, [ 'current', 'cancel' ] )
			? cancellableFetch.current.cancel()
			: noop()
	), [] );
```

Don't forget to add the dependencies if you agree with these changes.

Reply:
OK I'm not 100% sure on this but I've ended up using Optional Chaining as an alternative here.

Supported as of https://github.com/Automattic/wp-calypso/blob/c3608b5c2b656cad1209f52b0e2229838c2592ea/packages/calypso-build/CHANGELOG.md#510